README.md

```diff
@@ -25,7 +25,14 @@ steps.forEach(step => {
 
 To solve an equation:
 ```js
-const steps = mathsteps.solveEquation('2x + 2x + x + x');
+const steps = mathsteps.solveEquation('2x + 3x = 35');
+
+steps.forEach(step => {
+    console.log("before change: " + step.oldEquation.print());  // e.g. before change: 2x + 3x = 35
```

Comment:
Haha, okay! In that case, we gotta keep some count on how much character the README can take in before it goes for the next line so we don't put that many characters on a single line of code!